include/event2/listener.h

```diff
@@ -107,9 +107,18 @@ typedef void (*evconnlistener_errorcb)(struct evconnlistener *, void *);
  * code works as expected without affected by bindv6only sysctl setting in
  * system.
  *
- * This socket option also supported by Windows.
+ * This socket option on Windows is instead enabled by default.
  */
 #define LEV_OPT_BIND_IPV6ONLY		(1u<<8)
+/** Flag: Indicates that the listener wants to work only in both IPv4 and 
+ * IPv6 socket.
+ *
+ * This flag exists as copmlement to LEV_OPT_BIND_IPV6ONLY to account for
+ * the different default behaviour on Windows so that the code can
+ * explicitly request the socket to support both modes without having
+ * to rely on the default option.
+ */
+#define LEV_OPT_BIND_IPV4_AND_IPV6		(1u<<9)
```

Comment:
Let's enable this option explicitly if `LEV_OPT_BIND_IPV6ONLY` is not set

Reply:
i didn't want to forcefully enable it as then it could cause possible issues if we try to forcefully enable such option on a socket where the setsockopt could fail for a different reason and thus breaking already working code (for example forcefully setting it on a socket created with ``AF_UNSPEC``) and thus making the function fail.

Reply:
The idea was to make the behavior identical with Linux, but I'm ok with it as-is.